README.md

```diff
@@ -467,6 +467,14 @@ Manages the GPG keys that Apt uses to authenticate packages.
 
 This module is tested and officially supported on Debian 6 and 7 and Ubuntu 10.04, 12.04, and 14.04. Testing on other platforms has been light and cannot be guaranteed.
 
+This module is not designed to be split across stages. If you want to ensure that apt update is run before all packages, add a proper dependency like this:
+
+~~~puppet
+Class['apt::update'] -> Package<||>
+~~~
+
+Be aware that this conflicts with virtual package resources.
```

Comment:
I would like to use a different word than conflict. In Puppet lingo that's normally associated with resource conflicts, and that's not the case. It doesn't really conflict in that sense, it just causes all virtual resources to be realised at that point because of the unfortunate side-effect that collectors `<||>` cause realisation.